### PR TITLE
Remove "Console" options from Interface-only installer

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
The advanced tab of the Windows Interface-only installer had three "console" advanced install options. These related to a cut-down version of the Sandbox's Windows system tray icon, with only some of the menu items such as "GoTo" - but these currently don't work properly. The "console" were not enabled by default in the advanced installer and were not enabled in the express install.

This PR temporarily removes the three "console" options from the Interface-only advanced install options - until such time as the "console" is fixed (and perhaps renamed) or it's decided it should be removed permanently.

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
### Testing

Build 2 installers:
- Interface-only
- Interface + Sandbox

The Interface-only installer shoudl be smaller than the Interface + Sandbox installer.

Test Interface-only installer:
- Express install works:
  - Installs Interface.
  - Doesn't install domain server - domain-server.exe is not present in the install directory.
  - Interface runs.
- Custom install works:
  - There are no "Console" checkbox items present in the advanced options.
  - Installs Interface.
  - Doesn't install domain server - domain-server.exe is not present in the install directory.
  - Interface runs.

Test Interface + Sandbox installer:
- The 3 "Sandbox" checkbox items are present.
- Express install works as expected.
- Custom install works as expected. In parficular, the Sandbox server is installed or not installed, and run or not run, per checkbox items.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-10ff14a.tar_e2ffde22.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-10ff14a.tar_e2ffde22.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-10ff14a_e2ffde22.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-10ff14a_e2ffde22.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-10ff14a_e2ffde22.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-63ba953.tar_a3512d8a.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-63ba953.tar_a3512d8a.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-63ba953_a3512d8a.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-63ba953_a3512d8a.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-63ba953_a3512d8a.exe

**Comment by ctrlaltdavid** [comment]
Raised at a recent dev/community meeting: how to notify people if they have a message or something while Interface is minimized. The Console app has this sort of functionality. Perhaps the Console app should be fixed / improved (and perhaps renamed to something more fitting).

**Comment by kasenvr** [comment]
Will test this after https://github.com/kasenvr/project-athena/pull/651 is merged.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-3724219.tar_a3512d8a.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-3724219_a3512d8a.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-3724219.tar_a3512d8a.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-3724219_a3512d8a.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR682/Vircadia-Alpha-PR682-3724219_a3512d8a.exe

**Comment by kasenvr** [comment]
image
So the interface + sandbox install worked fine, but I guess I should have tried the Interface only first... still seems to show the checkboxes.

File: cmake/templates/NSIS.template.in [hunk 1]
```diff
@@ -836,16 +836,24 @@ Function PostInstallOptionsPage
...
+  ; ${If} @SERVER_COMPONENT_CONDITIONAL@
+  ;   ${NSD_CreateCheckbox} 0 $CurrentOffset$OffsetUnits 100% 10u "&Create a desktop shortcut for @SANDBOX_HF_SHORTCUT_NAME@"
+  ; ${Else}
+  ;   ${NSD_CreateCheckbox} 0 $CurrentOffset$OffsetUnits 100% 10u "&Create a desktop shortcut for @CONSOLE_HF_SHORTCUT_NAME@"
+  ; ${EndIf}
+  ;
+  ; Pop $DesktopConsoleCheckbox
+  ; IntOp $CurrentOffset $CurrentOffset + 15  
+  ; ; set the checkbox state depending on what is present in the registry
+  ; !insertmacro SetInstallOption $DesktopConsoleCheckbox @CONSOLE_DESKTOP_SHORTCUT_REG_KEY@ ${BST_UNCHECKED}
```
Comment (daleglass) [inline cmake/templates/NSIS.template.in:849]: If we're removing things, why is there a whole bunch of  commented out code being added? Did something go wrong during a merge?

**Review by daleglass — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Vircadia is a 3D social software project seeking to incrementally bring about a
 
 ### How to generate an Installer
 
-[For Windows](https://github.com/kasenvr/project-athena/blob/master/INSTALL.md)
+[For Windows](https://github.com/kasenvr/project-athena/blob/master/INSTALLER.md)
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply README.md:33]: Also in this PR, INSTALL.md is renamed to INSTALLER.md.

[the remark this answers is not in the document]
Comment (daleglass) [reply README.md:33]: Oops, missed that. Ok.

**Comment by ctrlaltdavid** [comment]
This PR is very old. I'm going to redo it from latest master.

**Pull request closed without merging**